test/slurm/job_conf.xml

```diff
@@ -26,7 +27,16 @@
           -->
           <param id="docker_volumes" from_environ="GALAXY_DOCKER_VOLUMES">$defaults</param>
         </destination>
+        <destination id="local" runner="local"/>
     </destinations>
+    <tools>
+        <!-- Tools can be configured to use specific destinations or handlers,
+             identified by either the "id" or "tags" attribute.  If assigned to
+             a tag, a handler or destination that matches that tag will be
+             chosen at random.
+         -->
+        <tool id="upload1" destination="local"/>
```

Comment:
Here you go. I suppose one could get enough stuff installed on the nodes to set the metadata and such there, but that's been more hassle than it's worth for me.

I haven't followed the PR that closely, so certainly some of this may no longer be needed.
[local.txt](https://github.com/bgruening/docker-galaxy-stable/files/328523/local.txt)